compute/elastic-metal/how-to/configure-flexible-ipv6.mdx

```diff
@@ -0,0 +1,120 @@
+---
+meta:
+  title: How to attach and detach a flexible IPv6 to and from an Elastic Metal server
+  description: This page explains how to attach and detach a flexible IPv6 to and from an Elastic Metal server
+content:
+  h1: How to attach and detach a flexible IPv6 to and from an Elastic Metal server
+  paragraph: This page explains how to attach and detach a flexible IPv6 to and from an Elastic Metal server
+dates:
+  validation: 2022-03-30
+  posted: 2022-03-30
+---
+
```

Comment:
Maybe we should add a **#How to attach a flexible IPv6 address** section, with a link back to the /compute/elastic-metal/how-to/configure-flexible-ip/ document? Or else change the title of this document to "how to configure a flexible IPv6..."

Could also be worth adding to the previous doc that there is no further configuration necessary for IPv4, but for IPv6 please see this document?